### DISCUSSION TITLE
2023-08-04

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#258] [daily_meeting]
### 오늘의 할 일
- 데모데이 발표
- 백엔드 재참여 로직 이슈 해결
- QA

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by aak2075** [comment]
### 체크인

10점 - 회식이 기대되네요

**Comment by nlom0218** [comment]
### 체크인

10점 - 발표 덕분에 매우매우 떨려요

**Comment by jaehee329** [comment]
### 체크인
7점 - 피곤혀요~

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 발표 빨리 끝내고 회식하러 가고 싶네요...

**Comment by woosung1223** [comment]
### 체크인
9점 - 발표 데모가 잘 되었으면 좋겠습니닫..

**Comment by yeopto** [comment]
### 체크인

10점 - 금요일이라서 기분이 좋아요
